docs/software/media.md

```diff
@@ -556,13 +556,20 @@ Also Installs:
     - Enter the details of your path (See the image example below to add your USB drive music folder to Ampache).
     - Select the `add catalogue` button.
 
-    Remark: For Ampache to access custom directories, you must ensure read access to the folder, e.g. by:
+    !!! note "Check access rights on local custom directories"
+        For Ampache to access custom directories, you must ensure read access to the folder, e.g. by:
 
-    ```sh
-    chmod -R 775 /my/directory
-    ```
+        ```sh
+        chmod -R 775 /my/directory
+        ```
 
-    ![Ampache web interface screenshot with instructions how to add media](../assets/images/dietpi-software-media-ampacheaddcatalogue.png){: width="400" height="180" loading="lazy"}
+        ![Ampache web interface screenshot with instructions how to add media](../assets/images/dietpi-software-media-ampacheaddcatalogue.png){: width="400" height="180" loading="lazy"}
+
+    !!! note "Access rights on remote mounts (e.g. Samba, NFS)"
+        In case that you want to access a mounted directory, also the access rights need to be set.  
+        Example Samba mount: Edit the `/etc/fstab` file and search for the line with the mounted media Samba share.  
+        Check whether `file_mode=0775,dir_mode=0775` is set.  
+        Otherwise set them and restart your system to get the changed mount options valid (alternatively execute `umount /my/directory` and `mount -a`).
```

Comment:
A reboot is not required, instead:
```sh
mount -o remount /my/directory
```
Also `mount -a` mounts all defined mounts, so while the above is a better choice to only apply new mount options, instead of mounting everything, `mount /my/directory` would be more targeted.

The changed modes make me think whether we should apply `0775` mode by default for Samba mounts with `dietpi-drive_manager`? Read access for all users is the default on Debian, additional execute permissions on all files for convenience, in case there are indeed executables present + `dietpi` group write access for media and download software which may download to, import or change metadata of media files.